src/EditorFeatures/Core/Shared/Extensions/GlyphExtensions.cs

```diff
@@ -211,5 +211,279 @@ public static ImageId GetImageId(this Glyph glyph)
                     throw new ArgumentException(nameof(glyph));
             }
         }
+
+        public static Glyph GetGlyph(this ImmutableArray<string> tags)
```

Comment:
> `this ImmutableArray<string>`

No. 😑

Reply:
Could you please explain more?

---
In reply to: [207998444](https://github.com/dotnet/roslyn/pull/29016#discussion_r207998444) [](ancestors = 207998444)

Reply:
Implement as not an extension method

---
In reply to: [208048322](https://github.com/dotnet/roslyn/pull/29016#discussion_r208048322) [](ancestors = 208048322,207998444)

Reply:
😆 Forgot to update here. Had a discussion offline - issue is we don't want to add a feature-specific extension method to a type which is used throughout the code base for many other purposes. Changing to a normal method fixes the issue.